README.md

```diff
@@ -42,6 +45,32 @@ echo "POSTGRESQL_PASSWORD='YourPassword'" >> .env
 echo "POSTGRESQL_USERNAME='YourUserName'" >> .env
 
 bin/rails db:setup
+bin/rails db:migrate
+bin/rails db:seed
+```
+
+## Testing
+
+### Test
+
+We use rspec framework to test our rails applicatio. To test you can run:
+
+```sh
+bundle exec rspec
+```
+
+### Jest
+
+To test our js files we use jest framework. Run with:
+
+```sh
+yarn test
+```
+
+and you change some component you should to update de snapshot with:
```

Comment:
this shouldn't be "and if you change"?

Reply:
Please change this to: `and if you change any components, you should update the snapshot with:`

PS: `de` 🦑